scripts/list_boards.py

```diff
@@ -17,7 +17,7 @@
 
 BOARD_SCHEMA_PATH = str(Path(__file__).parent / 'schemas' / 'board-schema.yml')
 with open(BOARD_SCHEMA_PATH, 'r') as f:
-    board_schema = yaml.safe_load(f.read())
+    board_schema = yaml.load(f.read(), Loader=yaml.CSafeLoader)
```

Comment:
I vaguely remember seeing a comment that this does not work on windows, have you tested this on windows to ensure it works?

Reply:
I will change this to the method used in `hardwaremap.py` and `scl.py`, where we only try to import the C versions and have a normal fallback. This will still give us speedups in CI, while being safer.